.gitattributes

```diff
@@ -1,4 +1,4 @@
-*.pbxproj -diff linguist-generated
+*.pbxproj text -merge union linguist-generated
```

Comment:
Is this something we would like to try?

https://github.com/lavajeff/react-native-apollo-starter/blob/825a3c7c61b0c357a5fd75dc48209f6b856a5f6d/.gitattributes#L2